Tests/HTTPHeadersTests.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  HTTPHeaders.swift
```

Comment:
`HTTPHeaders.swift` -> `HTTPHeadersTests.swift`.

Reply:
Fixed in [aaa3e6a](
https://github.com/Alamofire/Alamofire/pull/2629/commits/aaa3e6a10d2b14ad0dbc049dc5aa3b1b9bb8fa2e).